src/work-allocation-2/components/find-person/find-person.component.ts

```diff
@@ -14,26 +16,26 @@ import { FindAPersonService } from '../../services/find-person.service';
 export class FindPersonComponent implements OnInit {
   @Output() public personSelected = new EventEmitter<Person>();
   @Input() public title: string;
-  @Input() public domainString: string = 'BOTH';
+  @Input() public domainString: string = 'All';
   @Input() public findPersonGroup;
   @Input() public selectedPerson: string;
-  public domain: PersonDomain;
+  public domain: string;
```

Comment:
Can you explain why it is now a string please?

Reply:
as explained above